examples/grid/LayoutGrids.html

```diff
@@ -641,13 +641,13 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
             </ul>
           </td>
         </tr>
-        <tr>
+        <tr data-test-id="gridcell-role">
           <th scope="row"><code>gridcell</code></th>
           <td></td>
-          <td><code>span</code></td>
+          <td><code>span, div</code></td>
```

Comment:
I wanted to point on this minor change -- another place where the table was out of date with the examples. "tab-index" can be found on both span and div elements.

Reply:
I've addressed this in master.